src/routes/chat/Chat.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Moment from 'react-moment';
+
+import ChatList from './ChatList/ChatList.container';
+import ChatRoom from './ChatRoom/ChatRoom.container';
+import ChatInput from './ChatInput/ChatInput.container';
+import ChatOffer from './ChatOffer/ChatOffer';
+import SocketResponseService from './SocketService/socketResponseService';
+import './Chat.scss';
+
+SocketResponseService.initialize();
+Moment.startPooledTimer();
```

Comment:
!!! don't put it outside of the function. start it on onEffect. Then use onEffect's return to shut it down when the component unmounts. Same for Moment.